invenio_oauthclient/templates/invenio_oauthclient/settings/index.html

```diff
@@ -13,6 +13,8 @@
 
 {% set panel_title = _("Linked accounts") %}
 {% set panel_icon = "fa fa-link" %}
+{% set num_linked_services = services|selectattr("account")|list|length %}
+{% set can_disconnect = (num_linked_services > 1 or not local_login_impossible) %}
```

Comment:
The name of the var has changed to `only_external_login` if I am not mistaken:
```suggestion
{% set can_disconnect = (only_external_login and num_linked_services > 1) %}
```

The same below.

Reply:
indeed, nice catch!